Packs/CommonTypes/ReleaseNotes/3_3_97.md

```diff
@@ -0,0 +1,5 @@
+
+#### Indicator Types
+
+- **IPv6**
+Updated IPv6 regex for better performance and to avoid MAC addresses.
```

Comment:
Why did you new line ?

Reply:
Thats our styling guide
![image](https://github.com/demisto/content/assets/24267931/301aef13-5554-4a8c-9357-41db109deab2)

Reply:
👍